third_party/xla_client/xrt_computation_client.cc

```diff
@@ -1113,7 +1113,7 @@ std::unique_ptr<xrt::XLAComputation> XrtComputationClient::CreateXrtComputation(
         output_shape->ToProto();
   }
   *xrt_computation->mutable_hlo_snapshot() =
-      std::move(*computation.Snapshot().value());
+      std::move(*computation.Snapshot().ConsumeValueOrDie());
```

Comment:
It looks like this is breaking the build
```
tensorflow/compiler/xla/xla_client/xrt_computation_client.cc:1116:41: error: ‘class tensorflow::StatusOr<std::unique_ptr<xla::HloSnapshot> >’ has no member named ‘ConsumeValueOrDie’; did you mean ‘ValueOrDie’?
       std::move(*computation.Snapshot().ConsumeValueOrDie());
                                         ^~~~~~~~~~~~~~~~~
                                         ValueOrDie
```

Reply:
Thanks, let me look into this -- I was able to build this before my leave for the demo, but it might have something to do with the new tf update.